### PR TITLE
Blue bottle

### PR DESCRIPTION
**Pull request by billyfung**
for #48

scraper for Blue Bottle

### CODE REVIEW

File: scrapers/bluebottle.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+from bs4 import BeautifulSoup
+import requests
+from models import Coffee
+import logging
+
+# scraping Blue Bottle roasters
+
+def scrape_bluebottle():
+
```
Comment (jinpark) [inline scrapers/bluebottle.py]: why space

File: scrapers/bluebottle.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                # only works for not single origin
+                region = coffee_soup.find('p', {'class':'spec-details'}).contents[0].strip()
+            except AttributeError:
+                # for single origin, region is in name
+                # otherwise if espresso, no region
+                # grab only country for now
+                if 'Espresso' in name:
+                    region = ""
+                else:
+                    region = name.split()[0]
```
Comment (jinpark) [inline scrapers/bluebottle.py]: what if its `El Salvador` ?

File: scrapers/bluebottle.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            logging.info("Getting url: {}".format(url))
+            r = requests.get(product_url)
+            coffee_soup = BeautifulSoup(r.content)
+            active = True
+            price = float(coffee_soup.find('span', {'class':'js-variant-price'}).string[1:])
+            description = coffee_soup.find('p', {'class':'spec-overview'}).string
+            notes = coffee_soup.p.string.lower().split(',')
+            try:
+                # only works for not single origin
+                region = coffee_soup.find('p', {'class':'spec-details'}).contents[0].strip()
```
Comment (jinpark) [inline scrapers/bluebottle.py:40]: I dont see this in any of the pages

File: scrapers/bluebottle.py (hunk 2 above)
Comment (billyfung) [inline scrapers/bluebottle.py]: right... guess it doesn't work

File: scrapers/bluebottle.py (hunk 3 above)
Comment (billyfung) [inline scrapers/bluebottle.py:40]: see what? what pages?

Comment (jinpark) [inline scrapers/bluebottle.py:40]: I dont see the class `spec-details` anywhere on any page

**Comment by billyfung** [comment]
spec

File: scrapers/bluebottle.py (hunk 3 above)
Comment (billyfung) [inline scrapers/bluebottle.py:40]: spec

Comment (jinpark) [inline scrapers/bluebottle.py:40]: jin is an idiot. I seriously could not find it before

**Comment by jinpark** [comment]
lgtm

**Pull request merged**